view/template/developer/_quickstartCredits.php

```diff
@@ -19,7 +19,16 @@
   "name": "electricsheep",
   "file_path": "\\home\kauffj\\Desktop\\electric-sheep.mp4",
   "bid": 1,
-  "metadata":  { } <span class="code-bash__comment">//this should match the metadata returned by resolve_name </span>
+  "metadata":  {
+    "description": "Some interesting content",
+    "title": "my_interesting_content",
```

Comment:
Perhaps titles should be mainly formatted for humans? Like "My Interesting Content" or "My interesting content".